lib/components/util/glyph-icon.js

```diff
@@ -0,0 +1,29 @@
+import {Glyphicon} from 'react-bootstrap'
```

Comment:
Do we want to have a duplicate for `Icon` in the same folder? Are there icons in bootstrap that can't be found in font-awesome?